src/main/kotlin/racing/domain/car/Car.kt

```diff
@@ -0,0 +1,46 @@
+package racing.domain.car
+
+import racing.domain.exception.CarNameLengthExcessException
+
+data class Car(
+    private val _carName: String,
+    private var _moveCount: Int = 0
+) {
+    private var _positionByRound: MutableMap<Int, Int> = mutableMapOf()
+
+    init {
+        if (!CAR_NAME_RANGE.contains(_carName.trim().length)) {
+            throw CarNameLengthExcessException("자동차 이름은 ${CAR_NAME_RANGE.first}자 이상이어야 하고, ${CAR_NAME_RANGE.last}자를 초과할 수 없습니다.")
+        }
+    }
+
+    val carName: String
+        get() = _carName
+
+    val moveCount: Int
+        get() = _moveCount
+
+    val positionByRound: MutableMap<Int, Int>
+        get() = _positionByRound
```

Comment:
라이브 강의에서 배운 것처럼 Mutable 컬랙션을 드러낼땐 방어적 복사를 이용하여 Immutable 타입만 노출시켜보면 어떨까요~?
```suggestion
    val positionByRound: Map<Int, Int>
        get() = _positionByRound.toMap()
```
(만약 리스트로 바뀐다면 이런 느낌이 될 것 같아요. 예시니 참고만 해주세요!)

```suggestion
    val positions: List<Int>
        get() = _positions.toList()
```

Reply:
위 코멘트 참고해서 리스트로 바꿨고, 결국 요 형태로 구현이 되네요 😅